nmi_tools/glue/build/remote_task.pl

```diff
@@ -362,16 +363,8 @@ sub create_deb {
 }
 
 sub check_deb {
-    # Print package summary
-    # We would like to run lintian, but it is not available on all NMI machine
-    my $debs;
-    for (glob "*.deb") { if ($_ !~ /[+]sym/) { $debs .= $_ . " "; } }
-    if ( ! defined $debs) { $debs = "*.deb"; }
-    if ($ENV{NMI_PLATFORM} =~ /Debian8/) {
-        # Only works with a single deb
-        return "dpkg-deb -I $debs";
-    } else {
-        # return "lintian $debs";
-        return "dpkg-deb -W $debs";
-    }
+    # Run lintian on the packages.
+    # Add '--fail-on error' when Ubuntu20 is retired
+    # The ls can be dropped, when the errors are resolved.
+    return "lintian --jobs=$ENV{OMP_NUM_THREADS} *.changes; ls -lh *.deb";
```

Comment:
Is lintian available on all NMI machines now? Looking at removed comment from above

Reply:
Yes, Linitan is available in all Debian and Ubuntu NMI Docker images. (Otherwise, the check_native step would fail.)